src/proof/nova.rs

```diff
@@ -3655,4 +3655,29 @@ pub mod tests {
             None,
         );
     }
+
+    #[test]
+    fn test_issue424() {
```

Comment:
For consistency, let's call this `test_nested_letrec_regression`. Then you can put a reference to the issue number (which would be a new but welcome convention for these 'regression' tests) in a comment. Maybe you can change the `test_issue_426_minimal` just above similarly, while you're at it.